Add ExcludesFileReader framework

simionschiopu opened the pull request:
Remove all files related to excludes.yml parsing and add a ExcludesFileReader framework in project.

S2dentik commented on TaylorFramework/Modules/caprices/ProccessingMessage/Options/ExcludesFileOption.swift:
```diff
@@ -32,14 +33,13 @@ struct ExcludesFileOption: ExecutableOption {
+            return try ExcludesFileReader().absolutePathsFromExcludesFile(path, forAnalyzePath: analyzePath)
+        } catch FileReaderError.ExcludesFile(let errorMsg) {
             errorPrinter.printError(errorMsg)
             return nil
```
Removing this `return` will not lead to any changes

Commit:
Add ExcludesFileReader framework

Remove all files related to excludes.yml parsing and add a ExcludesFileReader framework in project.

simionschiopu commented:
Update
